mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -75,14 +76,33 @@ object CompletionValue:
     override def labelWithDescription(
         printer: MetalsPrinter
     )(using Context): String =
-      if symbol.is(Method) then s"${labelWithSuffix}${description(printer)}"
-      else if symbol.isConstructor then labelWithSuffix
-      else if symbol.is(Mutable) then
-        s"${labelWithSuffix}: ${description(printer)}"
+      if symbol.is(Method) then s"${label}${description(printer)}"
+      else if symbol.isConstructor then label
+      else if symbol.is(Mutable) then s"$label: ${description(printer)}"
       else if symbol.is(Package) || symbol.is(Module) || symbol.isClass then
-        if isFromWorkspace then s"${labelWithSuffix} -${description(printer)}"
-        else s"${labelWithSuffix}${description(printer)}"
-      else s"${labelWithSuffix}: ${description(printer)}"
+        if isFromWorkspace then
+          s"${labelWithSuffix(printer)} -${description(printer)}"
+        else s"${labelWithSuffix(printer)}${description(printer)}"
+      else if symbol.isType then s"${label}${description(printer)}"
+      else s"$label: ${description(printer)}"
+
+    def labelWithSuffix(printer: MetalsPrinter)(using Context): String =
+      if snippetSuffix.addLabelSnippet
+      then
+        def printTypeParam(ts: TypeSymbol): String =
+          val bounds =
+            ts.info match
+              case TypeBounds(lo, hi) =>
+                val loPart =
+                  if !lo.isNothingType then s" :> ${printer.tpe(lo)}" else ""
```

Comment:
I don't see methods like `isTrvial` or `isEmpty`, but maybe we could take a look at those in Scala 2 to fix some of the cases?

Reply:
Sorry, not sure what you mean. Could you elaborate?

Reply:
TypeBounds for Scala 2 have some additional methods, which maybe we could use not to show bounds if they are not needed

Reply:
Things like https://github.com/scala/scala/blob/344d99b7e1fadf81ccabe8badf5e4fb7296e98c4/src/reflect/scala/reflect/internal/Types.scala#L1545